src/algorithms/expected_consensus.md

```diff
@@ -197,10 +197,8 @@ Due to the existence of potential forks in EC, a miner can try to unduly influen
 
 This is detectable when a given miner submits two blocks that satisfy any of the following "consensus faults":
 
-- (1) `double-fork mining fault`: two blocks contains the same electionProof nonce, mined at the same height.
-- (2) `parent grinding fault`: one block's parent is a Tipset that could have validly included the other block according to Tipset validity rules, however the parent of the first block does not include the other block.
-
-  - While it cannot be proven that a miner omits known blocks from a Tipset in general (i.e. network latency could simply mean the miner did not receive a particular block) in this case it can be proven because a miner must be aware of a block they mined in a previous round.
+- (1) `double-fork mining fault`: two blocks mined off of the same Tipset at different epochs (i.e. with different `ElectionProof`s generated from the same input ticket).
+- (2) `parent grinding fault`: one block's parent is a Tipset that provably should have included a given block but does not. While it cannot be proven that a missing block was willfully omitted in general (i.e. network latency could simply mean the miner did not receive a particular block), it can when a miner has successfully mined a block two epochs in a row and omitted one. That is, this condition should be evoked when a miner omits their own prior block. When a miner's block at epoch e + 1 references a Tipset that does not include the block they mined at e both blocks can be submitted to prove this fault.
```

Comment:
you should write the code for this in `StoragePowerActorCode` (in a follow-up PR)